### ISSUE TITLE
statix check exits always with exit code 1

### ISSUE THREAD

**Issue by Gerschtli** [nerdypepper/statix#35]
Reproducible with the following steps:

```
$ nix build github:nerdypepper/statix
$ echo "{}" > file.nix
$ ./result/bin/statix check -- x.nix
$ ./result/bin/statix check --stdin < x.nix
```

The last two commands fail with exit code 1 and no output. I was not able to get a working output out of statix check so far..

Did I miss something or is this a bug?

**Comment by nerdypepper** [comment]
Oo, weird bug, thanks for the report, I have just pushed a fix to master, could you give that a go and let me know if it works?

**Comment by Gerschtli** [comment]
Yes, that fixed it! Thank you for your quick response :) Would love to see that fix released and updated in nixpkgs :)
